python-engine-test/src/test/java/io/deephaven/engine/table/impl/select/PythonMatchFilterTest.java

```diff
@@ -1,73 +0,0 @@
-/**
- * Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
- */
-package io.deephaven.engine.table.impl.select;
-
-import static org.junit.Assert.assertEquals;
-
-import io.deephaven.configuration.Configuration;
-import io.deephaven.engine.table.lang.QueryScope;
-import io.deephaven.io.log.LogLevel;
-import io.deephaven.io.logger.StreamLoggerImpl;
-import io.deephaven.util.process.ProcessEnvironment;
-import io.deephaven.engine.table.TableDefinition;
-import io.deephaven.engine.util.WorkerPythonEnvironment;
-import io.deephaven.jpy.PythonTest;
-
-import java.util.Collections;
-import java.util.List;
-
-import org.junit.Before;
-import org.junit.Ignore;
-import org.junit.Test;
-
-/**
- * Test MatchFilters that reference Python lists.
- */
-@Ignore // TODO (deephaven-core#734)
```

Comment:
https://github.com/deephaven/deephaven-core/issues/734 Says that someone needs to look into why these are failing before deciding to delete.  Was this analysis done?

Reply:
If this resolves all of #734, we should close that issue with this PR.